### PR TITLE
Entry for dlang

### PR DESCRIPTION
**Pull request by burner**
Provide link to graphql implementation in D

**[URL to the resource here.]**
https://github.com/burner/graphqld

**[Explain what this resource is all about and why it should be included here.]**
A link to the graphql implementation for the D programming language

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -390,6 +391,12 @@ for the Angel framework.
 * [graphql-client](https://github.com/tomhoule/graphql-client) - GraphQL client library for Rust with WebAssembly (wasm) support.
 * [graphql-parser](https://github.com/graphql-rust/graphql-parser) - A parser, formatter and AST for the GraphQL query and schema definition language for Rust.
 
+< name="lib-d" />
+
+### D (dlang) Libraries
+
+* [graphqld](http://github.com/burner/graphqld) - GraphQL server library for D.
```
Comment (addityasingh) [inline README.md]: I would add the `https` protocol

Comment (burner) [reply README.md]: thanks

**Comment by burner** [comment]
thank you for merging

**Comment by TreTuna** [comment]
That you for the pr!

**Pull request merged**